pkg/vm.go

```diff
@@ -35,6 +35,8 @@ type VMIface struct {
 	IP6DefaultGateway net.IP
 	// Private or public network
 	Public bool
+	// OriginalTapName without prefixes to help in decommissioning
+	OriginalTapName string
```

Comment:
You can always drive the tap name from the IP workload name. this is redundant and should not be stored here. All what u have to do in the decomission is just recompute the tap name from the reservation id .

Reply:
https://github.com/threefoldtech/zos/blob/master-3/pkg/primitives/vm.go#L312